sql/core/src/test/resources/sql-tests/results/show-tables.sql.out

```diff
@@ -60,58 +60,58 @@ SHOW TABLES
 -- !query schema
 struct<namespace:string,tableName:string,isTemporary:boolean>
 -- !query output
-show_t1
-show_t2
-show_t3
+	show_t3	true
```

Comment:
@yaooqinn so the golden file test framework also go through the Hive CLI code path?

Reply:
Yes, It use `hiveResultString`

Reply:
It did not go through the Hive CLI path
 - For `-- !query schema`, we use Dataframe.schema
 - For `-- !query output`, 
     - `SQLQueryTestSuite` uses hiveResultString directly
     - `ThriftServerQueryTestSuite` bypasses validation from 'show xxx' (cc @Yikf we shall enable these in this PR)

Reply:
Sure